rotors_gazebo_plugins/src/gazebo_imu_plugin.cpp

```diff
@@ -18,44 +19,61 @@
  * limitations under the License.
  */
 
+// MODULE HEADER
 #include "rotors_gazebo_plugins/gazebo_imu_plugin.h"
 
+// USER HEADERS
+#include "ConnectGazeboToRosTopic.pb.h"
+
+// SYSTEM LIBS
```

Comment:
The Google style guide suggests:
related header
system
3rd party
user

I have changed it to follow this.